theseus/geometry/tests/common.py

```diff
@@ -259,3 +275,19 @@ def exp_func(xi):
     ]
 
     assert torch.allclose(actual[0], expected, atol=atol)
+
+
+def check_projection_for_vector_log_map(tangent_vector, Group, atol=1e-8):
```

Comment:
We could reuse `check_projection_for_log_map()` but pass an extra kwarg that specifies if we should call `group.project()` before or not. I think that's the only difference, right?

Reply:
The `check_projection_for_map` is reused.